guides/common/modules/con_major-project-components-additional-resources.adoc

```diff
@@ -0,0 +1,5 @@
+[role="_additional-resources"]
+= Additional resources
```

Comment:
Again: I worry that the anchor is not really unique. I lean towards ".Additional resources".